File: 1341-split-a-string-in-balanced-strings/split-a-string-in-balanced-strings.go
func balancedStringSplit(s string) int {
    bal := 0;
    count := 0;
    for _, j := range s{
        if string(j) == "R"{
            bal++;
        }else{
            bal--;
        }
        if bal == 0{
            count++;
        }
    }
    return count;
}

File: 1341-split-a-string-in-balanced-strings/split-a-string-in-balanced-strings.py
class Solution:
    def balancedStringSplit(self, s: str) -> int:
        Bal = 0
        count = 0;
        for word in s:
            if word == 'R':
                Bal += 1;
            else:
                Bal -= 1;
            if Bal == 0:
                count += 1;

        return count;
        